### PR TITLE
Search bar functionality and styling

### LINKED ISSUE #17: Add a search bar and Implement a search feature with styling
**Is your feature request related to a problem? Please describe.**
Adding a search bar into the Nav bar and implementing a search bar feature with styling.

**Describe the solution you'd like**
Adding a search bar that can search different movies by putting an input in the search bar.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1.  Added a search bar
2. Styled the search bar

## Purpose
This allows user to search for movies

## Approach
Collected value from the search bar to make a new API call to display the search results. 

Closes #17

### CODE REVIEW

File: movie-app/src/header/NavBar.js [hunk 1]
```diff
@@ -1,26 +1,73 @@
...
+              id="input"
+              type="text"
+              className={searchOpen ? "search-bar" : "search-bar-close"}
+            ></input>
+            <button
+              type="button"
+              onClick={(e) => {
+                setSearchOpen(false);
+                clearText();
+              }}
```
Comment (gonzalespaulb) [inline movie-app/src/header/NavBar.js]: I suggest writing this function outside of the JSX element for better readability.

**Review by connielion — COMMENTED** [review]
Looks good.

**Review by gonzalespaulb — COMMENTED** [review]
Besides moving the function out of the search bar button everything else looks good.

File: movie-app/src/header/NavBar.js (hunk 1 above)
Comment (krferrell) [reply movie-app/src/header/NavBar.js]: That is a good idea. I will do that now

**Review by gonzalespaulb — APPROVED** [review]
LGTM

**Review by connielion — APPROVED** [review]
Good job on making code more readable in Navbar.js and input validation.

**Pull request merged**